.github/workflows/reusable_build_dev.yaml

```diff
@@ -31,7 +31,7 @@ on:
 jobs:
   build-and-test:
     # See https://github.com/actions/runner/issues/409#issuecomment-1158849936
-    runs-on: ${{ (inputs.arch == 'aarch64' && fromJSON('[ "self-hosted", "linux", "ARM64" ]')) || 'ubuntu-22.04' }}
+    runs-on: ${{ (inputs.arch == 'aarch64' && 'actuated-arm64-8cpu-16gb' || 'ubuntu-22.04' }}
```

Comment:
Do we know which OS is the `actuated` runner running?

Reply:
https://docs.actuated.dev/faq/#whats-in-the-vm-image-and-how-is-it-built it is designed to allow to replace `ubuntu-latest` with it for regular CI :)

Reply:
Hi folks, these machines are configured with ubuntu22.04

There's generally no need to use the container: feature and etcd removed it and got a performance boost.

Reply:
So, this should be the only occurrence where we were enforcing the `container: ubuntu-22.04`.
In other areas we need to enforce eg: fedora or centos7, thus we need to keep them!
Thanks @alexellis !

Reply:
Also, i opened a PR on libs to port circleci arm64 jobs to gha using `automated` runners: https://github.com/falcosecurity/libs/pull/1555
It is working flawlessly!

Reply:
Thanks @alexellis and @FedeDP ! I'll fix this PR and restart tests :)